boilerplate/app/components/button/button.story.tsx

```diff
@@ -4,6 +4,8 @@ import { storiesOf } from "@storybook/react-native"
 import { StoryScreen, Story, UseCase } from "../../../storybook/views"
 import { Button } from "./"
 
+declare var module
```

Comment:
Is this how you got around your whole `_module` problem?

Reply:
Yes, though I'm admittedly still a little confused about a few things:

- I don't understand the difference between this and the `global`definitions we've used before (and why only one central `global` definition works, but this declaration has to be duplicated in any file that uses `module`).
- I don't know why declaring it as a `global` caused a redbox error at runtime, since I thought Typescript was an entirely-compile-time thing.

... but at least this satisfies Typescript _and_ causes no runtime error.